src/app/input-sample/input-sample.component.html

```diff
@@ -158,7 +176,9 @@ <h3>Geolocation</h3>
     &lt;lux-input aria-label="Geolocation example" <span class="text-underlined">type="geolocation"</span>&gt;&lt;/lux-input&gt;</code>
     </pre>
     <div class="sample">
-        <lux-input aria-label="Geolocation example" type="geolocation"></lux-input>
+      <label>Geolocation: </label>
+      <lux-input aria-label="Geolocation example" type="geolocation" [(value)]="valueGeolocation"></lux-input>
```

Comment:
If you're going to add a label, remove the aria-label and add a `for` in the label. As it is currently, all the labels are orphan.

Reply:
Ok. I see. 
1. I need the label for standard users to read.
2. Can remove the aria-label then.
3. Will add a label.for + luxInput.inputId combo